sdk/storage/generate.ps1

```diff
@@ -1,16 +1,17 @@
 pushd $PSScriptRoot/Azure.Storage.Common/swagger/Generator/
 npm install
+npm install -g autorest@beta
```

Comment:
So local installations of autorest do not work? Did we file a bug about it?

Reply:
It seemed to still use the old core version when depending on the local dependency. I'm not sure that this is a bug, as you can specify the core version to use in the script. In any case, our Contributing.md recommends the global installation so it makes sense to be consistent here.